commands/misc/dl.js

```
@@ -0,0 +1,83 @@
+            return message.channel.send("You have to provide a URL in an argument.")
+        }
+        
+        const originalMessage = await message.channel.send("Downloading video...")
+        if(this.executeCommand(`yt-dlp "${url}" -P ${downloadsDir} --cookies ${cookieFilepath}`).error){
+            originalMessage.edit("An error occured when downloading the video.");
+            this.cleanUp(downloadsDir);
+            return;
+        }
+        
+
+        let files = fs.readdirSync(downloadsDir);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'files' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)